components/Card/Card.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './Card.module.css';
+
+interface Props {
```

Comment:
Используй в пропсах `название компонента + Props`, так немного проще воспронимать информацию, хотя это может быть субъективно, по крайней мере для консистентности
```suggestion
interface CardProps {
```